### PR TITLE
feat(graphql,graphene): add support for Python 3.11

### PR DESCRIPTION
**Pull request by Yun-Kim**
## Checklist
Blocked by #4854.

This change adds Python 3.11 testing to our graphql and graphene test suites.

- [x] Change(s) are motivated and described in the PR description.
- [x] Testing strategy is described if automated tests are not included in the PR.
- [x] Risk is outlined (performance impact, potential for breakage, maintainability, etc).
- [x] Change is maintainable (easy to change, telemetry, documentation).
- [x] [Library release note guidelines](https://ddtrace.readthedocs.io/en/stable/contributing.html#Release-Note-Guidelines) are followed.
- [x] Documentation is included (in-code, generated user docs, [public corp docs](https://github.com/DataDog/documentation/)).

## Reviewer Checklist

- [x] Title is accurate.
- [x] No unnecessary changes are introduced.
- [x] Description motivates each change.
- [x] Avoids breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
- [x] Testing strategy adequately addresses listed risk(s).
- [x] Change is maintainable (easy to change, telemetry, documentation).
- [x] Release note makes sense to a user of the library.

### CODE REVIEW

File: riotfile.py [hunk 1]
```diff
@@ -1791,8 +1791,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                     },
                 ),
                 Venv(
-                    pys=select_pys(min_version="3.6", max_version="3.10"),
-                    # FIXME[bytecode-3.11]: depends on bytecode module which doesn't yet support Python 3.11
+                    pys=select_pys(min_version="3.6", max_version="3.11"),
```
Comment (P403n1x87) [inline riotfile.py:1794]:
```suggestion
                    pys=select_pys(min_version="3.6"),
```
?

File: riotfile.py [hunk 2]
```diff
@@ -1812,8 +1811,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
...
```
Comment (P403n1x87) [inline riotfile.py:1814]:
```suggestion
                    pys=select_pys(min_version="3.6"),
```
?

Comment (Yun-Kim) [reply riotfile.py]: Brain fart 🤦

File: riotfile.py (hunk 1 above)
Comment (Yun-Kim) [reply riotfile.py]: Brain fart 🤦

**Review by emmettbutler — DISMISSED** [review]

**Review by mabdinur — CHANGES_REQUESTED** [review]
Can we add a release note?

**Comment by Yun-Kim** [comment]
> Can we add a release note?

After internal discussion, doesn't look like we'll need to add a release note since we never said in a previous release note we didn't support 3.11 for graphql/graphene.

**Review by brettlangdon — APPROVED** [review]
we could have included both notes in one release note file.

File: releasenotes/notes/feat-graphql-py-3-11-5431bd3eedb38c50.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    graphql: Adds support for Python 3.11.
```
Comment (mabdinur) [inline releasenotes/notes/feat-graphql-py-3-11-5431bd3eedb38c50.yaml]:
```suggestion
    graphql: Adds support for Python 3.11.
  - |
    graphene: Adds support for Python 3.11.
```

**Review by mabdinur — APPROVED** [review]

**Review by brettlangdon — APPROVED** [review]

**Pull request merged**